### PR TITLE
Travis integration

### PR DESCRIPTION
**Pull request by aravindavk**
- Fixed the golint issues reported by `gometalinter`
- Removed unused shell scripts

Signed-off-by: Aravinda VK <avishwan@redhat.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+script:
+  # Lint text-like files
+  - scripts/pre-commit.sh --require-all
+  # Fetch vendor dependencies so linters succeed
+  - make vendor-install
+  # Lint golang sources
+  - scripts/gometalinter-tests.sh
+  # Build container w/ Docker
+  # - docker version
+  # - ./build-docker-image.sh
```
Comment (shtripat) [inline .travis.yml]: This we will be enabling later. Right?

File: gluster-exporter/conf/conf.go [hunk 2]
```diff
@@ -22,13 +23,24 @@ type CollectorConfig struct {
...
 		msg := fmt.Sprintf("Failed loading config file: %s", confFilePath)
 		return msg
 	}
-	defer fileObj.Close()
-	data, _ := ioutil.ReadAll(fileObj)
+	defer func() {
+		err = fileObj.Close()
+		if err != nil {
+			// TODO: Log error
+			return
```
Comment (shtripat) [inline gluster-exporter/conf/conf.go]: We need an error message here. Anyway after we merge updated .toml based config, this function returns an error object and same can be used.

File: gluster-exporter/conf/conf.go [hunk 3]
```diff
@@ -42,13 +54,23 @@ func LoadCollectorsConfig(confFilePath string) string {
...
```
Comment (shtripat) [inline gluster-exporter/conf/conf.go]: Same as above.

File: gluster-exporter/main.go [hunk 4]
```diff
@@ -95,33 +97,37 @@ func main() {
...
+		_, _ = fmt.Fprintf(os.Stderr, "No Metrics registered, Exiting..\n")
 		os.Exit(1)
 	}
 
-	metricsPath, _ := conf.SystemConfig["metrics-path"]
-	port, _ := conf.SystemConfig["port"]
-	nport, _ := strconv.Atoi(port)
+	metricsPath := conf.SystemConfig["metrics-path"]
+	port := conf.SystemConfig["port"]
+	nport, err := strconv.Atoi(port)
```
Comment (shtripat) [inline gluster-exporter/main.go]: With toml based conf Atoi wont be required.

File: .travis.yml (hunk 1 above)
Comment (aravindavk) [reply .travis.yml]: Yes, I will remove this now and add this along with `Dockerfile` PR.

File: gluster-exporter/conf/conf.go (hunk 2 above)
Comment (aravindavk) [reply gluster-exporter/conf/conf.go]: we need log here, error message returned from defer will not reach the caller.

File: gluster-exporter/main.go (hunk 4 above)
Comment (aravindavk) [reply gluster-exporter/main.go]: ack

**Comment by aravindavk** [comment]
@shtripat  I rebased the PR after toml config PR merge. Please review

**Comment by aravindavk** [comment]
Thanks @shtripat

**Pull request merged**